contents/handbook/engineering/support-hero.md

```diff
@@ -146,7 +142,11 @@ We use [Zendesk Support](https://zendesk.com/) as our internal platform to manag
 
 Zendesk allows us to manage all our customer conversations in one place and reply through Slack or email. We also use [Help](https://www.atlassian.com/software/halp) if you want to easily create and manage Zendesk tickets directly from inside the PostHog Users Slack. 
 
-This ensures that we don't miss anyone, especially when their request is passed from one person to another at PostHog, or if they message us over the weekend. You should have received an invite to join both Zendesk and Help as part of onboarding - if you didn't, ask Grace. 
+This ensures that we don't miss anyone, especially when their request is passed from one person to another at PostHog, or if they message us over the weekend. You should have received an invite to join both Zendesk and Help as part of onboarding - if you didn't, ask Grace.
+
+Zendesk will get populated with new issues from people outside the PostHog organization on the `posthog` and `posthog.com` repos, and also Squeak questions. These tickets will come with links to the issue or Squeak so you can answer them in the appropriate platform, rather than on Zendesk itself.
+
+The feedback box on the PostHog app will automatically create a Zendesk ticket and let the user know that we've create a ticket for them. If it's a feature request feel free to reply with something short like "Thanks for the feedback! We'll take a look at this and get back to you if we have any questions." and pass it on to the team. If there's not enough information it might be worth reviewing the session recording or asking them for more information.
```

Comment:
```suggestion
The feedback box on the PostHog app will automatically create a Zendesk ticket and let the user know that we've create a ticket for them. If it's 'I love PostHog, it's great" just respond with an appropriate thank you. If it's a feature request ask them to file an issue in our open source GitHub repo. If there's not enough information it might be worth reviewing the session recording or asking them for more information.
```

Reply:
I'm going to merge in what we currently have as the point of the feedback box is to surface some of the smaller things that are too high friction to create a github issue for